maven_iuvs/search.py

```diff
@@ -321,6 +320,9 @@ def get_file_version(orbit_number, data_directory,
                            segment=segment,
                            channel=channel,
                            data_directory=data_directory)
+        # TODO: The above call returns a list of strings, but the 
```

Comment:
Actually, this whole function is gross because it doesn't operate on a file at all but goes to disk to look at what's there. I don't think this function is called by anything else in the repo, recommend deletion.

Reply:
Must have been used by Kyle? But now they have a whole other repository. Anyway, I actually think we should leave it for now. This falls under the issue #50 and I think it should have a separate PR (i.e. we can work on it later). that ok with you?